### ISSUE TITLE
Accessibility - 2.4.6 Heading and labels

### ISSUE THREAD

**Issue by phucduong86** [shiffman/shiffman.net#88]
Results for: https://shiffman.net

Findings: Label missing for email input. Recommend using aria-label
```
<input id="user-email" type="email" value="" name="EMAIL" class="form" placeholder="Your email">
```
